docs: k8s updates for the 1.5.0 release (#21329)
hashicorp/consul: website/content/docs/k8s/helm.mdx

    image that is used for functionality such as catalog sync.
    This can be overridden per component.

  - `imagePullPolicy` ((#v-global-imagepullpolicy)) (`string: ""`) - The image pull policy used globally for images controlled by Consul (consul, consul-dataplane, consul-k8s, consul-telemetry-collector).

>     One of "IfNotPresent", "Always", "Never", and "". Refer to https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy

Reviewer: ```suggestion
    One of "IfNotPresent", "Always", "Never", and "". See the Kubernetes [Image pull policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) for details.
```

    or may not be broadly accessible depending on your Kubernetes cluster.
    Set this to false to skip syncing ClusterIP services.

>   - `syncLoadBalancerEndpoints` ((#v-synccatalog-syncloadbalancerendpoints)) (`boolean: false`) - If true, LoadBalancer service endpoints instead of ingress addresses will be synced to Consul. 
>     If false, LoadBalancer endpoints are not synced to Consul.

Reviewer: ```suggestion
  - `syncLoadBalancerEndpoints` ((#v-synccatalog-syncloadbalancerendpoints)) (`boolean: false`) - If true, LoadBalancer service endpoints instead of ingress addresses are synced to Consul. 
    If false, LoadBalancer endpoints are not synced to Consul.
```
Nit: present tense for both options

              - external-dns.alpha.kubernetes.io/hostname
          ```

      - `metrics` ((#v-connectinject-apigateway-managedgatewayclass-metrics)) - Metrics settings for gateways created with this gateway class configuration.

        - `enabled` ((#v-connectinject-apigateway-managedgatewayclass-metrics-enabled)) (`boolean: -`) - This value enables or disables metrics collection on a gateway, overriding the global gateway metrics collection settings.

        - `port` ((#v-connectinject-apigateway-managedgatewayclass-metrics-port)) (`int: null`) - This value sets the port to use for scraping gateway metrics via prometheus, defaults to 20200 if not set. Must be in the port
          range of 1024-65535.

>         - `path` ((#v-connectinject-apigateway-managedgatewayclass-metrics-path)) (`string: null`) - This value sets the path to use for scraping gateway metrics via prometheus, defaults to /metrics if not set.

Reviewer: ```suggestion
        - `path` ((#v-connectinject-apigateway-managedgatewayclass-metrics-path)) (`string: null`) - This value sets the path to use for scraping gateway metrics via Prometheus, defaults to /metrics if not set.
```
